modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,6 +73,18 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // Redirecting user login page.
+    $userRolesArray = [
+      'administrator',
+      'site_owner',
```

Comment:
@kairamkondarajesh `site_owner` is a Y Cloud specific role and should not be here. You'll need to provide a way for this array to be supplemented by other code, or provide a comment that suggests that it be patched in case new roles need to be added.